README.md

```diff
@@ -399,7 +427,10 @@ The CallbackManager uses a timeout mechanism to ensure callbacks do not hang ind
 const customTimeout = 10; // 10 seconds
 const callbackManager = new CallbackManager(callbacks, customTimeout);
 ```
-
+```python
+custom_timeout = 10; # 10 seconds
+callback_manager = CallbackManager([my_logging_callback], custom_timeout)
+```
```

Comment:
Can you add link to your video in readme? I'll move it after

Reply:
Added raw link in beggining of section, ctrl-f "video: "